pages/_app.tsx

```diff
@@ -21,7 +20,6 @@ export default class App extends NextApp {
       <ViewerProvider>
         <MDXProvider components={shortCodes}>
           <CacheProvider value={cache}>
-            {globalStyles}
```

Comment:
I removed global styles since Tailwind handles that, but unsure whether `CacheProvider` is still needed.